node/tests/sync_test.go

```diff
@@ -112,3 +112,104 @@ func TestSyncStartStopLightWithBridge(t *testing.T) {
 
 	assert.EqualValues(t, h.Commit.BlockID.Hash, sw.GetCoreBlockHashByHeight(ctx, 40))
 }
+
+/*
+Test-Case: Sync a Full Node with a Bridge Node
+Steps:
+1. Create a Bridge Node(BN)
+2. Start a BN
+3. Check BN is synced to height 20
+4. Create a Full Node(FN) with a trusted peer
+5. Start a FN with a defined connection to the BN
```

Comment:
If I understand the code correctly here, that same FN has the BN as a trusted peer (and hence has a connection to the BN). The comment makes it sound like it is connected to a trusted peer and additionally to a (different) BN.

Reply:
True. I'll redo the comments as it sound redundant and confusing later 😓

Reply:
I've found other mentions like that in older tests, so I'll push the changed test-case comment section in a separate PR to not explode this with off-topic commits 🙏